config/routes.rb

```diff
@@ -107,7 +107,9 @@
   end
 
   namespace :super_admin do
-    resources :libraries
+    resources :libraries do
+      resource :image, only: [:edit, :update]
```

Comment:
Did you actually implement a controller for this? I don't see it.

Reply:
I copy/pasted the `images_controller` from admin. I'll add that back in... not sure how it didn't make it in. Maybe it should be extracted out into a shared directory for both the items and libraries to access?

Reply:
Is it required? What does it do?